src/main/java/uk/gov/hmcts/cmc/claimstore/services/ClaimReissueService.java

```diff
@@ -0,0 +1,62 @@
+package uk.gov.hmcts.cmc.claimstore.services;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.cmc.claimstore.events.claim.CitizenClaimCreatedEvent;
+import uk.gov.hmcts.cmc.claimstore.events.claim.PostClaimOrchestrationHandler;
+import uk.gov.hmcts.cmc.claimstore.events.solicitor.RepresentedClaimCreatedEvent;
+import uk.gov.hmcts.cmc.claimstore.idam.models.User;
+import uk.gov.hmcts.cmc.domain.models.Claim;
+
+import static uk.gov.hmcts.cmc.domain.models.ClaimState.CREATE;
+
+@Service
+public class ClaimReissueService {
+
+    private final Logger logger = LoggerFactory.getLogger(this.getClass());
+
+    private final ClaimService claimService;
+    private final UserService userService;
+    private final PostClaimOrchestrationHandler postClaimOrchestrationHandler;
+
+    public ClaimReissueService(
+        ClaimService claimService,
+        UserService userService,
+        PostClaimOrchestrationHandler postClaimOrchestrationHandler
+    ) {
+        this.claimService = claimService;
+        this.userService = userService;
+        this.postClaimOrchestrationHandler = postClaimOrchestrationHandler;
+    }
+
+    public void getCreatedClaimsAndReIssue() {
+        try {
+            User user = userService.authenticateAnonymousCaseWorker();
+            String authorisation = user.getAuthorisation();
+            claimService.getClaimsByState(CREATE, userService.authenticateAnonymousCaseWorker()).forEach(claim -> {
```

Comment:
user user instead of calling userService to authenticate anonymous user.

Reply:
sorry @karthikkannankainos didn't get it?

Reply:
You created a `user` by calling userService.authenticateAnonymouseCaseWorker. Then in the call to claimService.getClaimByState, you could pass that as variable than making the call again to user again.